server/src/main/java/no/ntnu/klubbhuset/domain/User.java

```diff
@@ -32,10 +35,13 @@ public void addSecurityGroup(SecurityGroup securityGroup) {
     @Id
     private String email;
 
-    @JsonIgnore
-    @JsonbTransient
     private String password;
 
+    @JsonbTransient
```

Comment:
Just curious, when you place the annotation "@JsonbTransient" on a method, does that inable the jsonbuilder serializing the password?

Reply:
Think so

Reply:
Ja